### PR TITLE
remove generated file data/python_parser.py, and add demo target in Makefile

### PR DESCRIPTION
**Pull request by mw66**
https://github.com/we-like-parsers/pegen/issues/61

### CODE REVIEW

**Comment by mw66** [comment]
@MatthieuDartiailh

**Review by MatthieuDartiailh — APPROVED** [review]
Look reasonable to me but MAKEFILE are not my forte so I would like a second pair of eyes before merging.

We could also have a changelog entry.

File: Makefile [hunk 1]
```diff
@@ -74,3 +74,8 @@ gh-pages:  ## Publish documentation on BBGitHub Pages
 .PHONY: help
 help:  ## Print this message
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+demo:
+	PYTHONPATH=$(shell pwd)/src:$(PYTHONPATH) python -m pegen data/python.gram -o data/python_parser.py
+	PYTHONPATH=$(shell pwd)/src:$(PYTHONPATH) python data/python_parser.py -r tests/demo.py
```
Comment (lysnikolaou) [inline Makefile:80]:
```suggestion
	PYTHONPATH=$(shell pwd)/src:$(PYTHONPATH) $(PYTHON) -m pegen data/python.gram -o data/python_parser.py
	PYTHONPATH=$(shell pwd)/src:$(PYTHONPATH) $(PYTHON) data/python_parser.py -r tests/demo.py
```

**Review by lysnikolaou — COMMENTED** [review]
Left one suggestion. Other than that, LGTM!

**Comment by MatthieuDartiailh** [comment]
Since you added a description to the readme a changelog entry would not help that much so merging. Thanks for your contribution.

**Pull request merged**